### PR TITLE
feat: 938 Update project card with sell info

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: regen-network/regen-registry#398

image

- Update ProjectCard so it can display the price per ton and credits available.

```
      purchaseInfo={{
        sellInfo: { creditsAvailable: '1,200', pricePerTon: '17.20-24.20 ' },
      }}
```

- Also, refactor the card a bit by splitting its code.
image
- add new `xxs` size for text

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed
- [ ] once the PR is closed, set up backport PRs for `redwood` and `hambach` (see below)

#### Setting up backport PRs

After merging your PR to `master`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR, `@Mergifyio backport redwood hambach`.

2. If your branch does have merge commits:

    a. Pull latest `master`, `hambach` and `redwood` branches

    b. Create new branches for backports and merge `master` (replace `<PR#>` with your PR #)
    ```
    git checkout -b hambach-backport-<PR#> hambach
    git merge master
    git push origin hambach-backport-<PR#>
    git checkout -b redwood-backport-<PR#> redwood
    git merge master
    git push origin redwood-backport-<PR#>`
    ```

    c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.
  

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by victordelval — APPROVED** [review]
Awesome! your work is a very good reference!

File: web-components/src/components/cards/ProjectCard/ProjectCard.styles.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import { Theme } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+
+export const useProjectCardStyles = makeStyles((theme: Theme) => ({
+  separator: {
+    border: `0.5px solid ${theme.palette.grey[100]}`,
+    [theme.breakpoints.up('sm')]: {
+      // marginTop: theme.spacing(6.25),
```
Comment (mhagel) [inline web-components/src/components/cards/ProjectCard/ProjectCard.styles.ts:8]: ✂️ ?

File: web-components/src/components/cards/ProjectCard/ProjectCard.styles.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      marginRight: theme.spacing(5.25),
+      marginLeft: theme.spacing(5.25),
+    },
+    [theme.breakpoints.down('sm')]: {
+      // marginTop: theme.spacing(4.5),
```
Comment (mhagel) [inline web-components/src/components/cards/ProjectCard/ProjectCard.styles.ts:13]: ✂️ ?

**Review by mhagel — APPROVED** [review]

**Review by haveanicedavid — APPROVED** [review]
looks good. I like the theme tweaks + pulling things out into util files

**Comment by mhagel** [comment]
@mergifyio v4 redwood hambach

**Pull request merged**